tests/testthat/test-families.R

```diff
@@ -0,0 +1,46 @@
+context("general family tests")
+
+test_that("test that all combinations run without errors", {
+
+  library(glmnet)
+  glmnet.control(fdev = 0)
+
+  n <- 1000
+  p <- 2
+  set.seed(1)
+
+  grid <- expand.grid(
+    family = c("gaussian", "binomial", "multinomial"),
+    intercept = TRUE,
```

Comment:
Is there a reason we aren't testing both TRUE and FALSE here? The issue in #2 only comes up when sparse=TRUE, right?